### ISSUE TITLE
Mazda 3, 2018

### ISSUE THREAD

**Issue by Betu009** [Trevelopment/MZD-AIO#157]
How can I update the map of my country in my Mazda 3 SD card or an USB drive?

**Comment by Spikeophant** [comment]
I don't think this project does map updates, you usually need to buy those from Mazda.  You can use Mazda toolbox to install them.

**Comment by Naoue55** [comment]
Les mises à jour se font gratuitement par l'intermédiaire de l'application "MazdaToolbox"

**Comment by Spikeophant** [comment]
you can get it from Mazda directly: https://mazda.welcome.naviextras.com/how-to-update.php
